### DISCUSSION TITLE
line-graph in mermaid-js?

### DISCUSSION THREAD

**Discussion by ghost** [mermaid-js/mermaid#3522] [Q&A]
Hi everyone.

I would like to know if you all have thought about adding line-graph to mermaid-js. *I'd like to help with the mermaid-js project and I was wondering if you've thought about integrating rechart-js for this with mermaid-js.*

#### Image
![image](https://user-images.githubusercontent.com/111701513/192260129-c272c4b5-eaee-4bfe-a28d-9df3788a10fc.png)

**source-image** [storytellingwithdata\line_graph](https://www.storytellingwithdata.com/blog/2020/3/24/what-is-a-line-graph)

#### Idea
```markdown
title: Produce Sales
subtitle: IN THOUSANDS (USD)
sales: [80, 70, 60, 50, 40, 30, 20, 10, 0]
month: [['OCT', '2019'], 'NOV', 'DEC', ['JAN', '2020'], 'FEB', 'MAR', 'APR', 'MAY', 'JUN']
```

**Comment by jgreywolf** [comment]
Closing as duplicate of #2532 (since it is currently assigned)

**Comment by sidharthv96** [comment]
https://github.com/mermaid-js/mermaid/issues/4269
